### PR TITLE
fix: Do not import Lumo twice for the document

### PR DESCRIPTION
**Pull request by Artur-**
All Lumo modules except utility are loaded and injected into <head> by https://github.com/vaadin/flow-components/blob/main/vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/Lumo.java We should not do that another time

This does not guarantee that the utility classes are injected before other styles but they are not meant to be overridden either

### CODE REVIEW

**Comment by Artur-** [comment]
This depends on  #16230 for the test

**Comment by Artur-** [comment]
Scrap that, it does not need that test nor does it yet resolve #16231. That needs another PR

File: flow-server/src/main/resources/plugins/application-theme-plugin/theme-generator.js [hunk 1]
```diff
@@ -186,15 +172,13 @@ function generateThemeFile(themeFolder, themeName, themeProperties, options) {
       imports.push(`import { ${lumoImport} } from '@vaadin/vaadin-lumo-styles/${lumoImport}.js';\n`);
     });
 
-    if (useDevServer) {
-      lumoImports.forEach((lumoImport) => {
+    lumoImports.forEach((lumoImport) => {
+      if (lumoImport === 'utility') {
```
Comment (mshabarov) [inline flow-server/src/main/resources/plugins/application-theme-plugin/theme-generator.js:176]:
```suggestion
      // Lumo is injected to the document by Lumo itself, except for the utility module
      if (lumoImport === 'utility') {
```

**Review by mshabarov — CHANGES_REQUESTED** [review]
Any test covering this patch?

**Comment by Artur-** [comment]
All existing tests that use Lumo pass so they should test that Lumo still works. I would not add a test that Lumo is not loaded twice because that tests the code and not any feature. 

The real problem that should be resolved is #16231, which has a disabled test in #16241. It can be fixed more easily once the test and this PR have been merged. It is an attempt to keep PRs reasonably sized

**Review by mshabarov — APPROVED** [review]

**Comment by Artur-** [comment]
Do we need this for 24.0? I think the theme loading changes that are needed are not in general suitable for a bugfix release

**Comment by mshabarov** [comment]
> Do we need this for 24.0? I think the theme loading changes that are needed are not in general suitable for a bugfix release

Since this patch fixes what was recently added to 24.0, I assumed 24.0 deserves this fix, especially that the codebase is almost equivalent and this won't require any additional maintenance effort from us.

**Pull request merged**